app/src/main/res/layout/fragment_criterions.xml

```diff
@@ -1,145 +1,156 @@
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@id/gender_selection_title">
-
-            <RadioButton
-                android:id="@+id/radioFemale"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:fontFamily="@font/fredoka_normal"
-                android:paddingStart="5dp"
-                android:paddingEnd="5dp"
-                android:text="@string/radio_female"
-                android:textSize="16sp" />
-
-            <RadioButton
-                android:id="@+id/radioMale"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:checked="false"
-                android:fontFamily="@font/fredoka_normal"
-                android:paddingStart="5dp"
-                android:paddingEnd="5dp"
-                android:text="@string/radio_male"
-                android:textSize="16sp" />
-
-            <RadioButton
-                android:id="@+id/radioMaleAndFemale"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:checked="true"
-                android:fontFamily="@font/fredoka_normal"
-                android:paddingStart="5dp"
-                android:paddingEnd="5dp"
-                android:text="@string/radio_male_and_female"
-                android:textSize="16sp" />
-
-        </RadioGroup>
-
-    </androidx.constraintlayout.widget.ConstraintLayout>
-
-    <Button
-        android:id="@+id/criterionButtonDone"
-        android:layout_width="match_parent"
-        android:layout_height="0dp"
-        android:text="@string/meetup_creation_meetup_done"
-        app:layout_constraintTop_toBottomOf="@+id/gender_selector"
-        tools:layout_editor_absoluteX="185dp" />
-
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingStart="20dp"
+        android:paddingEnd="20dp"
+        android:paddingTop="20dp"
+        android:paddingBottom="20dp"
+        android:orientation="vertical"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="@string/criterions_title"
+            android:fontFamily="@font/fredoka_bolder"
+            android:textColor="@color/theme_dark"
```

Comment:
Should use ?attr to refer to color, so that it dynamically changes according to the users selected theme